Britannia/common/on_action/BTWK_death.txt

```diff
@@ -0,0 +1,27 @@
+﻿on_death = {
+	on_actions = {BTWK_uther_dies_on_action}
+}
+
+
+#Trigger to return Arthur to Dumnonia a few weeks after Uther's death (simulate time for news to reach Arthur, & for his return)
+
+BTWK_uther_dies_on_action = {
+	effect = {
+		if = {
+			limit = {
+				trigger_if = {
+					limit = {
+						root = character:1
+						global_var:uther_is_dead = 0
```

Comment:
Instead of binary variables, it's usually better to check for the presence or absence of a variable (e.g. has_global_variable = X to check, and set_global_variable = X/remove_global_variable = X to toggle).